source/daplink/RTX_Config.c

```diff
@@ -90,7 +90,7 @@
 #elif defined(INTERFACE_SAM3U2C)
 #define OS_CLOCK    96000000
 #elif defined(INTERFACE_LPC4322)
-#define OS_CLOCK    204000000
+#define OS_CLOCK    96000000
```

Comment:
Can you provide details on the speed the lpc4322 can stably run for your configuration based on the voltage it is running at?